analyzer/src/main/kotlin/managers/SpdxDocumentFile.kt

```diff
@@ -98,16 +98,23 @@ private fun String.extractOrganization(): String? =
         line.withoutPrefix(SpdxConstants.ORGANIZATION)
     }.firstOrNull()
 
+/**
+ * Return whether the string has the format of an [SpdxExternalDocumentReference], with or without an additional
+ * package id.
+ */
+private fun String.isExternalDocumentReferenceId(): Boolean =
+    substringBefore(":").startsWith(SpdxConstants.DOCUMENT_REF_PREFIX)
```

Comment:
Nit: The `substringBefore(":")` does not seem to be necessary here as it's implied by the `startsWith()` check. Do you still want to keep it for clarify? (Which would be fine with me.)

Reply:
I prefer it without the unnecessary.